### PR TITLE
update some translation on 'kubectl zh_CN'

### PR DESCRIPTION
**Pull request by cnfatal**
I found a unfriendly translation when using kubectl. so i improve it, and update some other translation incidentally.

### CODE REVIEW

**Comment by stewart-yu** [comment]
Why commit the Pr in this repo? will check later

**Comment by fatalc** [comment]
Modified related documents according to the contribution guide.

**Comment by zhangxiaoyu-zidif** [comment]
assign @zjj2wry

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -966,37 +966,37 @@ msgid ""
 "\t\tkubectl run pi --schedule=\"0/5 * * * ?\" --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'"
 msgstr ""
 "\n"
-"\t\t# Start a single instance of nginx.\n"
+"\t\t# 启动一个单实例(single instance)的 nginx.\n"
 "\t\tkubectl run nginx --image=nginx\n"
 "\n"
-"\t\t# Start a single instance of hazelcast and let the container expose port 5701 .\n"
+"\t\t# 启动一个单实例的 hazelcast 并 暴露容器的 5701 端口.\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: remove extra spaces

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 2]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run hazelcast --image=hazelcast --port=5701\n"
 "\n"
-"\t\t# Start a single instance of hazelcast and set environment variables \"DNS_DOMAIN=cluster\" and \"POD_NAMESPACE=default\" in the container.\n"
+"\t\t# 启动一个单实例的 hazelcast 并 设置容器的环境变量 \"DNS_DOMAIN=cluster\" 和 \"POD_NAMESPACE=default\"\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: remove extra spaces

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 3]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
-"\t\t# Start a replicated instance of nginx.\n"
+"\t\t# 启动一个多副本的 nginx.\n"
 "\t\tkubectl run nginx --image=nginx --replicas=5\n"
 "\n"
-"\t\t# Dry run. Print the corresponding API objects without creating them.\n"
+"\t\t# Dry run. 打印相应的API对象而不创建它们.\n"
 "\t\tkubectl run nginx --image=nginx --dry-run\n"
 "\n"
-"\t\t# Start a single instance of nginx, but overload the spec of the deployment with a partial set of values parsed from JSON.\n"
+"\t\t# 启动一个单实例的 nginx, 但是使用从JSON解析的部分值集合重载部署的规范(spec).\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 但是使用从 JSON 解析的部分值集合覆盖 deployment 的 spec 信息

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 4]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run nginx --image=nginx --overrides='{ \"apiVersion\": \"v1\", \"spec\": { ... } }'\n"
 "\n"
-"\t\t# Start a pod of busybox and keep it in the foreground, don't restart it if it exits.\n"
+"\t\t# 启动一个 busybox pod 并将其保留在前台运行,不自动重启.\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 符号后添加空格

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 5]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run -i -t busybox --image=busybox --restart=Never\n"
 "\n"
-"\t\t# Start the nginx container using the default command, but use custom arguments (arg1 .. argN) for that command.\n"
+"\t\t# 使用default命令启动nginx容器， 但使用自定义参数(arg1 .. argN) .\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: ditto

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 6]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run nginx --image=nginx -- <arg1> <arg2> ... <argN>\n"
 "\n"
-"\t\t# Start the nginx container using a different command and custom arguments.\n"
+"\t\t# 使用不同的命令和自定义参数启动nginx容器.\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: ditto

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 7]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run nginx --image=nginx --command -- <cmd> <arg1> ... <argN>\n"
 "\n"
-"\t\t# Start the perl container to compute π to 2000 places and print it out.\n"
+"\t\t# 启动perl容器以计算π到2000位并将其打印出来.\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: ditto

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 8]
```diff
@@ -966,37 +966,37 @@ msgid ""
...
 "\t\tkubectl run pi --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'\n"
 "\n"
-"\t\t# Start the cron job to compute π to 2000 places and print it out every 5 minutes.\n"
+"\t\t# 启动cron作业以计算π到2000位并每5分钟打印一次.\n"
```
Comment (zjj2wry) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: ditto

**Comment by zjj2wry** [comment]
@fatalc thank you do this，I want to push this pr to be merged, leaving some comments，would like fix this

**Comment by fatalc** [comment]
@zjj2wry translation improved , let me know if any problem.😋

**Comment by fatalc** [comment]
@zjj2wry PR rebased, please take a look  and RE-APPROVED 👌

**Comment by fatalc** [comment]
it shoud be closed , i think

**Pull request closed without merging**